src/Routes/Register/Helpers.js

```diff
@@ -533,36 +534,61 @@ const schema = (intl) => ({
       name: 'rhui-last-part',
       label: (
         <React.Fragment>
-          <Title headingLevel="h3" size="md">
-            {intl.formatMessage(messages.configureBasicAuthTitle)}
-          </Title>
           <TextList component={TextListVariants.ol}>
             <TextListItem>
-              {intl.formatMessage(messages.configureBasicAuthStep1)}
-            </TextListItem>
-            <TextListItem>
-              {intl.formatMessage(messages.configureBasicAuthStep2)}
-            </TextListItem>
-            <TextListItem>
-              {intl.formatMessage(messages.configureBasicAuthStep3)}
+              {intl.formatMessage(messages.configureActivationKeysStep1, {
+                hybridCloudConsole: (
+                  <a href="https://console.redhat.com/insights/connector/activation-keys">
```

Comment:
Would be great if we could build a link according to environment: so users are navigated to console.stage.redhat.com/... if we work in stage.

Also, must it be open in a new tab? If not, then we have to use `Link` or `InsightsLink` in order to prevent the whole page to reloaded.

Reply:
Also, just a reminder, we are soft-deprecating `intl`, you can put the text strings directly into JSX without calling `formatMessage`